src/ncp/ncp_base.cpp

```diff
@@ -41,14 +41,25 @@
 #include <platform/radio.h>
 #include <platform/misc.h>
 
-namespace Thread {
+namespace Thread
+{
 
 extern ThreadNetif *sThreadNetif;
 
 static NcpBase *sNcpContext = NULL;
 
 #define NCP_PLAT_RESET_REASON        (1<<31)
 
+enum
+{
+    kThreadModeTLV_Receiver    = (1 << 3),
+    kThreadModeTLV_Secure      = (1 << 2),
+    kThreadModeTLV_DeviceType  = (1 << 1),
+    kThreadModeTLV_NetworkData = (1 << 0),
+};
+
+#define UNDEFINED_RSSI      (-128)
```

Comment:
I've gone ahead and made this change, as well as adding a comment that this value is limited to being used with the PROP_MAC_WHITELIST command.